wg-releases/wg-qa/meeting-notes/2019-9-18.md

```diff
@@ -0,0 +1,79 @@
+# Post-Summit: QA Working Group
+
+Meeting date: Wed Sept 18 @1:00pm-PST
+Attendees: 
+
+* Link to pre-summit session notes:
+    * [CI, Build, & Test pre-summit](https://hackmd.io/LJZ9D222RlulmL5clYisqA)
+* Link to summit session notes:
+    * [Testing session](http://bit.ly/ems-testing)
+    * [CI & Build](http://bit.ly/ems-ci)
+* Link to summit session photos:
+    * https://photos.app.goo.gl/eHpRPxJ3wcLH87uk6
+
```

Comment:
Can we delete this from our meeting notes? The governance repo is readable to the general public and we don't want these summit links to be public. The rest of the notes look OK for public.

Reply:
Done!